src/App.js

```diff
@@ -2,18 +2,15 @@ import "bootstrap/dist/css/bootstrap.min.css";
 import "./App.css";
 import Footer from "./components/Footer";
 import Header from "./components/Header";
-import {
-  default as HornedBeast,
-  default as HornedGallery,
-} from "./components/HornedBeast";
+import { default as HornedBeast } from "./components/HornedBeast";
```

Comment:
For this line here, you can import HornedBeast without curlies since you have exported this as the default inside the component already 👍 

- So `import HornedBeast from 'filepath here'`;
- If you have time, read more on export default vs exports  (curlies are used with exports and with default export we can just import component as is.